src/ocean/math/SlidingAverage.d

```diff
@@ -478,7 +478,7 @@ public class SlidingAverageTime ( T ) : SlidingAverage!(T)
 
     ***************************************************************************/
 
-    public T opAddAssign ( T val )
+    public T opOpAssign ( string op ) ( T val ) if (op == "+")
```

Comment:
It might be worth to have more in depth unit tests for this change. At the moment it seems to just instantiate it.

Reply:
Needs to update documentation params too